Extensions/easy_video_links.js

```diff
@@ -0,0 +1,69 @@
+
+	run: function() {
+		
+		this.running = true;
+		XKit.tools.init_css("easy_video_links");
+		XKit.interface.create_control_button("xkit-easy-video-links-button", this.button_icon, "Copy Video Address", "");
```

Comment:
the fourth argument here should be a function that gets called when the button is clicked (equivalent to your document on click handler below)

Reply:
My advice: get rid of the `init` function and instead have a `copyLink` function that serves as your click handler.